### PR TITLE
add list locales tool

### PR DESCRIPTION
**Pull request by mfs12**

### CODE REVIEW

File: Tools/list-locales/UI/UserInterfaceConstants.hpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#ifndef USER_INTERFACE_CONTANATS
```
Comment (wilriker) [inline Tools/list-locales/UI/UserInterfaceConstants.hpp:1]: Typo. Not important though.

**Review by wilriker — APPROVED** [review]
There is one typo though not important, so OK for me to merge if you are fine with leaving this typo.

File: Tools/list-locales/UI/UserInterfaceConstants.hpp (hunk 1 above)
Comment (mfs12) [reply Tools/list-locales/UI/UserInterfaceConstants.hpp]: fixed.

**Comment by dc42** [comment]
What was the reason for changing ecv.h? That is a standard header file, used to support static analysis using Escher C Verifier.

**Comment by wilriker** [comment]
It's a copy reduced to the bare-minimum, not a modification of the original.

**Comment by dc42** [comment]
Yes; but why was it changed at all?

**Comment by mfs12** [comment]
@dc42 double checked it. After improving the includes this hack is not needed. Howto proceed? Revert the merge? Create another pull request?

**Comment by mfs12** [comment]
created a new merge request with a fix for the header include. #103.

**Comment by dc42** [comment]
It's not important right now because eCv needs to be extended to support lambda functions before it can be used on PanelDueFirmware again; but best fix it anyway. I suggest another PR, since the merge affected other files too.

**Pull request merged**